src/gc/unix/gcenv.unix.cpp

```diff
@@ -530,6 +532,9 @@ uint32_t GCToOSInterface::GetCurrentProcessCpuCount()
     if (count == 0 || count > 64)
         count = 64;
 
+    if (cgroup.GetCpuLimit(&cgroupCpuCount)){
```

Comment:
@gboucher90 This code doesn't build by default, you can run `build.cmd <arch> <config> buildstandalonegc` to test it and see if it builds.